src/main/resources/application.properties

```diff
@@ -18,7 +18,7 @@ mail.debug=true
 mail.smtp.auth=true
 mail.main.URL=http://localhost:8080
 mail.activationURL=${mail.main.URL}/users/confirmation?emailUUID=
-mail.resetPasswordURL=${mail.main.URL}/reset-password?emailUUID=
+mail.resetPasswordURL=http://localhost:4200/reset-password?emailUUID=
```

Comment:
Please create separate property called `mail.ui.URL` and assign to it `http://localhost:4200`
Then use it value here

Reply:
Yes, I agree about an ugly code but I want to make it better